### PR TITLE
CLI

### PR DESCRIPTION
**Pull request by cryptoquick**
Issue: #7 
Added an initial project structure.
This is mostly to skeletonize the project so we can add more meat to the commands.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,12 +18,12 @@
 setup(
     name="hashd",
     version=version,
-    description="A fully decentralized broadcast system for the based on gossip and proof of work",
```
Comment (cryptoquick) [inline setup.py]: @Harryman Just so you know, that is copied from the README. I'll let you change it there.

**Comment by Harryman** [comment]
I think we could fill out the functions from componentFunctions.py, they should be pretty PoC ready in themselves. We should wrap the functions in a module though so we can reuse instance variables instead of passing from function to function.

**Comment by cryptoquick** [comment]
@Harryman sounds good, I'll get on that

File: setup.py (hunk 1 above)
Comment (Harryman) [reply setup.py]: ah, didn't catch it in the README.md, i'll just edit here in github

**Pull request merged**